zerver/openapi/testing.yaml

```diff
@@ -0,0 +1,136 @@
+test1:
+  responses:
+    '200':
+       content:
+         application/json:
```

Comment:
Do we want to add a test case to this using additionalProperties?  I think we don't have one today.